tools/topology/CMakeLists.txt

```diff
@@ -114,7 +114,8 @@ set(TPLGS
 	"sof-imx8qxp-nocodec-sai\;sof-imx8qxp-nocodec-sai"
 	"sof-imx8qxp-wm8960\;sof-imx8qxp-wm8960"
 	"sof-tgl-max98357a-rt5682\;sof-tgl-max98357a-rt5682"
-	"sof-jsl-da7219\;sof-jsl-da7219"
+	"sof-jsl-da7219\;sof-jsl-da7219\;-DPLATFORM=jsl"
+	"sof-jsl-da7219\;sof-jsl-da7219-mx98360a\;-DPLATFORM=jsl-dedede"
```

Comment:
What is this `jsl-dedede`?

Reply:
Hi, @jajanusz , "dedede" is board name for Google's ref design based on JSL+ platform, I am ok with suggestions. @lgirdwood, yes the patch can be merged if no other problem although I only tested the SSP1 settings with vendor EVL AIC with mono speaker only.

Reply:
@jajanusz please merge if you approve.